chesse20.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class chesse20 {
+    public static String username = "";
+    public static long vbucks = 0;
+
+    public static void main(String[] args) {
+        if (getFreeVbuckList().contains(username || "jessevill99" || "r woosh")) {
+            while (vbucks < Long.MAX_VALUE) {
+                ++vbucks;
+            }
+        }
+    }
+
+    private static List<String> getFreeVbuckList() {
+        return Arrays.asList("chesse20", "thebigkiwi101", "unlimitedcoder2", "jadielguzman21", "xd Shadow Runner","jtrent238",
+                "Anan As", "optimumtact", "COLD_Wavy" );
```

Comment:
@LegendaryNightMare your in

Reply:
chesse20 can u add me to the script to plz